docs/source/development.md

```diff
@@ -97,3 +97,35 @@ coverage3 html
 ```
 
 and then open htmlcov/index.html or any other reports of interest.
+
+## Docs
+
+To build the docs, first install Sphinx. If on macOS, you can install via brew, so long as you add sphinx-doc to
+your path afterwards:
+
+```
+brew install sphinx-doc
+echo 'export PATH="/usr/local/opt/sphinx-doc/bin:$PATH"' >> ~/.zshrc
+source ~/.zshrc
```

Comment:
i think it's easier using `pip install`

Reply:
I tried the `pip install` approach and couldn't get it working. Maybe we can pair on this tonight after the team meeting?